### PR TITLE
Support motion control, added fullscreen switch.

### PR DESCRIPTION
**Pull request by elin-moco**
The motion control signal is jumpy anyway...

### CODE REVIEW

File: public/js/client.js [hunk 1]
```diff
@@ -16,12 +18,12 @@ var pos = {x: 0, y: 0, cmd: null};
  * {integer} pos.y mouse y offset
  * {string} pos.cmd key command or mouse click (not implemented yet)
  */
-var emitMouse = function(x, y, cmd) {
-  pos.x = x;
-  pos.y = y;
-  pos.cmd = cmd;
+var emitMouse = function (x, y, cmd) {
+    pos.x = x;
```
Comment (gasolin) [inline public/js/client.js]: You can run `gulp lint` to check if all styles are settled.

hint: keep `2 spaces` as indent, and don't put space before brace arguments.

File: public/js/client.js [hunk 2]
```diff
@@ -30,49 +32,112 @@ var emitMouse = function(x, y, cmd) {
...
+            y = 90
+        }
+        if (y < -90) {
+            y = -90
+        }
+        x += 90;
+        y += 90;
+        emitMouse(x, y, 'motion')
+    }
+});
```
Comment (gasolin) [inline public/js/client.js]: please keep a blank line at the end to pass the style check

**Comment by elin-moco** [comment]
Tested on Android and iOS,
Currently Android works best for motion control (much smoother motion signal),
but can't use touch in fullscreen mode.
Motion event in Firefox OS is blocked when orientation locked, 
and the motion signals are unstable (using Flame).

**Comment by gasolin** [comment]
Looks good to me, thanks!

**Pull request merged**